### PR TITLE
#128 Add tests for components

### LINKED ISSUE #128: [Chore]: Add unit tests to components
Add unit tests to following components:
- [x] background
- [x] button
- [ ] card
- [ ] connect-wallet-btn
- [ ] header
- [x] input
- [ ] modals
- [x] not-available
- [x] textarea
- [ ] tooltip
- [ ] transaction

### PR DESCRIPTION
**Pull request by farebord**
https://github.com/Taraxa-project/hype/issues/128

Please link the issue:

Fixes #128 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.
- [ ] The changes impact front-end.

## For changes impacting frontend

Please leave some screenshots/video recordings of the newly implemented interface and the Figma link for its initial design.

### Screenshots/ recordings

=> Multimedia here.

### Figma link

=> Figma link here.

## Description

Please provide a short description about what the PR contains.

## Funny gif

Leave a funny gif for faster review.

### CODE REVIEW

File: services/client/src/testUtils.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { FC, ReactElement } from 'react';
+import { render, RenderOptions } from '@testing-library/react';
+import { HypeThemeProvider } from './theme/HypeTheme';
+
+const AllTheProviders: FC<{ children: React.ReactNode }> = ({ children }) => {
```
Comment (Malak67) [inline services/client/src/testUtils.tsx:5]: @farebord Why do we need this file? Are we using it?
I don't see it used anywhere. Maybe it is left from the wagmi tests.

**Review by Malak67 — APPROVED** [review]
LGTM! Left one minor comment.

!!! Please pull develop and fix merge issues before merging this branch.

**Review by VargaElod23 — CHANGES_REQUESTED** [review]
Please fix conflicts and add the test runs into the workflow when changes are made to the frontend like it is for the contract unit tests.

File: services/client/src/testUtils.tsx (hunk 1 above)
Comment (farebord) [reply services/client/src/testUtils.tsx:5]: It's been used on every test. It's a wrapper that adds hypetheme to it.
`import { render, screen } from 'src/testUtils';`

Check imports.

**Review by VargaElod23 — APPROVED** [review]

**Pull request merged**